openio/openio.go

```diff
@@ -192,12 +195,37 @@ func collectMetax(ns string, service string, proxyURL string, c chan netdata.Met
 			}
 		} else if s[1] == "volume" {
 			go volumeInfo(service, ns, s[2], c)
-		} else if s[0] == "gauge" {
-			// TODO: do something with gauge?
 		}
 	}
 }
 
+type metaxInfoBody struct {
+	Cache map[string]int64
+	Elections map[string]int64
+}
+
+func collectMeta2Info(ns, service, proxyURL string, c chan netdata.Metric) {
+	url := fmt.Sprintf("http://%s/v3.0/forward/info?id=%s", proxyURL, service)
+	sid := util.SID(service, ns)
+	info := metaxInfoBody{}
+
+	res, err := util.HTTPGet(url)
+
+	if err != nil {
+		// TODO handle err
```

Comment:
👍

Reply:
In this case, there is not so much you could do. It is better to return and skip.
We could  eventually log it in DEBUG mode.